src/Transforms/BottomUp.dfy

```diff
@@ -644,8 +644,8 @@ module Bootstrap.Transforms.Proofs.BottomUp_ {
 
   // TODO: e and e' should be the same actually
   lemma EqInterp_Expr_UnaryOp_CanBeMapLifted(
-    e: Interp.Expr, e': Interp.Expr, op: UnaryOp, v: WV, v': WV
-  )
+      e: Interp.Expr, e': Interp.Expr, op: UnaryOp, v: WV, v': WV
+    )
```

Comment:
Is this on purpose?

Reply:
Yes. Argument lists are indented by 2 spaces in this formatter, and since arguments are inside, they are indented by 4 spaces. This also helps differentiating arguments from bodies. But I'll add this to my final survey.